tests/baselines/reference/constructorOverloads4.errors.txt

```diff
@@ -23,7 +23,7 @@ tests/cases/compiler/constructorOverloads4.ts(11,1): error TS2348: Value of type
     
     (new M.Function("return 5"))();
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-!!! error TS2349: Cannot invoke an expression whose type lacks a call signature.
+!!! error TS2349: Cannot invoke an expression whose type lacks a call signature. Type 'Function' has no compatible call signatures.

```

Comment:
Can you explain the semantics here? I don't think that `class Function` and `function Function` will merge (hence the duplicate identifier errors), so the error that changes should be based on only one of the definitions. The first definition has no call signatures, so if the error is based on the first definition, then it looks correct to me.